src/components/Table/Table.tsx

```diff
@@ -51,6 +54,18 @@ export const TableRow: FC<PropsWithChildren<TableRowProps>> = ({
   )
 }
 
+const FIRST_COLUMN_WIDTH = "4fr"
+const MIDDLE_COLUMN_WIDTH = "1fr"
+const LAST_COLUMN_WIDTH = "3.5rem"
+
+function generateGridColumns(numColumns?: number): string {
+  const nbColumns = numColumns === undefined || numColumns < 3 ? 6 : numColumns
+  const middleColumnsWidths = Array(nbColumns - 2)
+    .fill(MIDDLE_COLUMN_WIDTH)
+    .join(",")
+  return `lg:grid-cols-[${FIRST_COLUMN_WIDTH},${middleColumnsWidths},${LAST_COLUMN_WIDTH}]`
```

Comment:
You can't do this with tailwind. Styles will only be compiled for classes that exist in plaintext in the source code. I was thinking you'd just accept fixed values for `numCols` like `5 | 6` and manually handle it with `clsxm`. 

The actual problem is that `TableRow` is using `clsx` when it should be using `clsxm`!

`clsxm` uses [tailwind-merge](https://github.com/dcastil/tailwind-merge) to ensure there are never classname conflicts. 

I pushed a commit with a resolution.